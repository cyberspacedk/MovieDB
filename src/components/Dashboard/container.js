import React from 'react';
import Header from '../Header';

const Dashboard = () => {
  return (
    <div>
      <Header />
    </div>
  );
};

export default Dashboard;
