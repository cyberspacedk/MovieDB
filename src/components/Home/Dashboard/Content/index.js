import ContentContainer from './container';

export default ContentContainer;
