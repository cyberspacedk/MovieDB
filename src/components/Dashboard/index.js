import Dashboard from './container';

export default Dashboard;
