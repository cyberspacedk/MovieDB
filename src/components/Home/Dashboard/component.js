import React from 'react';
import { Switch, Route } from 'react-router-dom';
import Header from './Header';
import Content from './Content';
import Movie from '../../Movie';

const Dashboard = () => {
  return (
    <div>
      <Header />
      <Switch>
        <Route exact path="/" component={Content} />
        <Route path="/:id" component={Movie} />
      </Switch>
    </div>
  );
};

export default Dashboard;
