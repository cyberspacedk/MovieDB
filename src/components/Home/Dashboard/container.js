import React from 'react';
import Header from './Header';
import SearchPanel from './Content/SearchMovies';
import Content from './Content';

const Dashboard = () => {
  return (
    <div>
      <Header />
      <SearchPanel />
      <Content />
    </div>
  );
};

export default Dashboard;
