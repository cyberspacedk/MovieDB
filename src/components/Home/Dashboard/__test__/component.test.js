import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../component';

describe('Component: Dashboard', () => {
  it('Match its snapshot. Display Dashboard', () => {
    const container = shallow(<Dashboard />);
    expect(container).toMatchSnapshot();
  });
});
