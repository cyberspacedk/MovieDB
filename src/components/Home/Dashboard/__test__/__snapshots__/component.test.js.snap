// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Dashboard Match its snapshot. Display Dashboard 1`] = `
<div>
  <PageHeader />
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(PageContent)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(MyListsContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/lists/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(FavoritesContainer)",
          "type": [Function],
        }
      }
      path="/favorites"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(WatchListContainer)",
          "type": [Function],
        }
      }
      path="/watchlist"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(ListDetailsContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/lists/:id"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Movie)",
          "type": [Function],
        }
      }
      path="/:id"
    />
  </Switch>
</div>
`;
